Packs/CheckpointFirewall/Integrations/CheckPointFirewall_v2/CheckPointFirewall_v2.yml

```diff
@@ -1758,6 +1758,13 @@ script:
       name: session_timeout
       required: false
       secret: false
+    - default: false
+      defaultValue: ''
```

Comment:
add also parameter `domain`.
Currently you added only `domain` arg to the `checkpoint-login-and-get-session-id` command